drop current style guide and replace with CDN version of MDL, index updated

codeandrop opened the pull request:
Based on issue #674 , these are the changes in this pull request:
- Drop the current style guide and Sass sources
- Replace them in the index.html with CDN references to the latest version of MDL
- Update the index.html layout to use the MDL text template (from 'Templates' on getmdl.io)
- SASS and JS File updated

The only question I have is if styleguide.html should be deleted as well, I can do that in this PR or in another if necessary.

Thanks

addyosmani commented:
Whoa. Only just saw this :) Thanks @codeandrop! I would say drop the current styleguide.html. We *could* keep it around if we were doing something like http://feditor.io/content/gist/a0234ef382c8ea0c903f to preview components, but I'm unsure what the overall value would be to end users.

sgomes commented on app/index.html:
```diff
@@ -33,37 +33,243 @@
     -->
 
     <!-- Page styles -->
+    <link rel="stylesheet" href="https://storage.googleapis.com/code.getmdl.io/1.0.2/material.indigo-pink.min.css">
```
We may want to add some brief documentation here explaining how to choose your own colours.

codeandrop replied:
done

codeandrop commented:
You're welcome @addyosmani ! Agree with the styleguide.html I removed it.
Also I added link to the MDL customization page and brief documentation based on @sgomes suggestion.
Let me know if this works  for you.

addyosmani commented:
LGTM to merge. 

@sgomes @surma @nicolasgarnier if you have a chance next week, would you mind doing a sweep through the project README and docs just to make sure that we're in line with what's present here? I'm fine with merging and then tackling those next. I'd consider this a prio for MDL once 1.0.3 is looking in good shape :shaved_ice:

surma commented:
Sure thing :dancer:

surma commented:
@addyosmani Not sure if I understood you right, but the README of WSK looks fine, esp. because there is no mention of MDL. Maybe there should be?

Also, should we update the browser support listing?
